### ISSUE TITLE
New builds (v1.0.1) Slewlimiter bug fixed

### ISSUE THREAD

**Issue by martin-lueders** [martin-lueders/Befaco#3]
[Befaco-1.0.1-mac.zip](https://github.com/martin-lueders/Befaco/files/3318563/Befaco-1.0.1-mac.zip)
[Befaco-1.0.1-win.zip](https://github.com/martin-lueders/Befaco/files/3318555/Befaco-1.0.1-win.zip)
[Befaco-1.0.1-lin.zip](https://github.com/martin-lueders/Befaco/files/3318567/Befaco-1.0.1-lin.zip)

**Comment by Eurikon** [comment]
any idea when this will be integrated in to VCV Rack ?

**Comment by martin-lueders** [comment]
No. I send the pull request to Andrew several weeks ago. I will try to
contact him again, to see what the status is.

Cheers,
Martin
